### PR TITLE
compleseus: improve performance by delaying preview

### PR DESCRIPTION
**Pull request by TheJJ**
- vertico is on elpa now
- adjust preview delay so the preview's startup (loading major mode, fontifying) isn't triggered instantly when selecting a candiate, but rather when lingering on it for a short time.

### CODE REVIEW

**Review by lebensterben — APPROVED** [review]

**Comment by sebhahn** [comment]
is this pull request fixing this issue as well https://github.com/doomemacs/doomemacs/issues/7064?

**Comment by TheJJ** [comment]
yea, that's how I even noticed :)

**Comment by sebhahn** [comment]
Can this be merged soon? Without these changes it is not possible to get a buffer list

File: layers/+completion/compleseus/packages.el [hunk 1]
```diff
@@ -36,24 +36,7 @@
     (selectrum :toggle (eq compleseus-engine 'selectrum))
     (vertico
      :toggle (eq compleseus-engine 'vertico)
-     ;; TODO remove when `vertico-repeat' on ELPA
```
Comment (sebhahn) [inline layers/+completion/compleseus/packages.el:39]: why did you remove the extension of vertico, they are not an elpa or are they?

Comment (TheJJ) [reply layers/+completion/compleseus/packages.el:39]: they are - the the extension files are part of the https://elpa.gnu.org/packages/vertico.html tar archive.

Comment (sebhahn) [reply layers/+completion/compleseus/packages.el:39]: thanks! it seems that they are not enabled by default, is it possible to add them again?

**Comment by lebensterben** [comment]
fixes #15941

**Comment by lebensterben** [comment]
thx

**Comment by cremacs1** [comment]
removing all of the vertico extensions from `compleseus-packages` made them all inaccessible, they seem to need to be listed. E.g. `compleseus/init-vertico-directory` doesn't work right now, but if you add `vertico-directory` back, it does.

**Comment by sebhahn** [comment]
Yes, I noticed it too and added them back. However, there has been a discussion how to deal with the exentions https://github.com/minad/vertico/issues/83 and it seems they are stuck in a subfolder called "./extensions". Installing vertico from MELPA doesn't add/download the subfolder, but if it would be possible to download them too there are possibilities to load them e.g. using straight or adding them to the load-path

**Comment by TheJJ** [comment]
Hmm, the ELPA archive contains the files directly in the archive root directory, not in a subdirectory, but the git repo contains it in a subdir. The previous hack by defining new packages directly from single files out of the git repo now has to be replaced by something better, the vertico-directory with the module files is in `load-path`, one just needs to load the files.

**Comment by sebhahn** [comment]
the extensions were missing for me, but actually deleting the vertico folder helped, since vertico is now downloaded again including the extension

@thejj do you have an example/config where to put the load-path so that the existing init functions work in the compleseus packages.el, thanks!

**Comment by TheJJ** [comment]
I've submitted #15945 which should now fix the extension loading :)
